3party/protobuf/CMakeLists.txt

```diff
@@ -66,6 +66,7 @@ endif()
 if (PLATFORM_WIN AND MSVC)
   set(
     SRC
+    ${SRC}
```

Comment:
These set(A ${A} ...) looks ugly. Can we use cmake lists here instead? The expression above will look like:
```cmake
list(APPEND SRC src/google/...)
```